src/main/java/dev/railroadide/logger/impl/DefaultLogger.java

```diff
@@ -138,6 +141,52 @@ public void log(String message, LoggingLevel level, Object... objects) {
         loggingMessages.offer(message);
     }
 
+    private String arrangeMessage(LoggingLevel level, String message){
+        LocalTime localTime = LocalTime.now();
+
+        long hours = localTime.getHour();
+        long minutes = localTime.getMinute();
+        long seconds = localTime.getSecond();
+        long milliseconds = localTime.getNano() / 1_000_000;
+        long nanoseconds = localTime.getNano();
+
+        String threadName = Thread.currentThread().getName();
+        String loggingLevelName = level.name();
+        String loggerName = this.name;
+
+        //arrange using config file
+        try {
+            if(!Files.exists(this.configFile)){
+                //default config
+                Files.writeString(this.configFile, "{hours}:{minutes}:{seconds} [{threadName}] {loggingLevelName} {loggerName} - {message}", StandardOpenOption.CREATE_NEW);
+            }
+
+            String logMessageArrangement = Files.readString(this.configFile);
+            Map<String, Object> m = new HashMap<>();
+
+
+            m.put("{hours}", hours);
+            m.put("{minutes}", minutes);
+            m.put("{seconds}", seconds);
+            m.put("{milliseconds}", milliseconds);
+            m.put("{nanoseconds}", nanoseconds);
+
+            m.put("{threadName}", threadName);
+            m.put("{loggingLevelName}", loggingLevelName);
+            m.put("{loggerName}", loggerName);
+            m.put("{message}", message);
+
+
+            for(Map.Entry<String, Object> entry :  m.entrySet()){
+                logMessageArrangement = logMessageArrangement.replace(entry.getKey(), String.valueOf(entry.getValue()));
+            }
+
+            return logMessageArrangement;
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
```

Comment:
All the variables can be inlined. Saves 9 lines of code, since the variables arent used anywhere else.